<ui version="4.0" >
<comment> Begin CVS Header </comment>
<comment>   $Source: /Volumes/Home/Users/shoops/cvs/copasi_dev/copasi/UI/CQModelValue.ui,v $ </comment>
<comment>   $Revision: 1.11 $ </comment>
<comment>   $Name:  $ </comment>
<comment>   $Author: shoops $ </comment>
<comment>   $Date: 2009/11/30 19:34:16 $ </comment>
<comment> End CVS Header </comment>

<comment> Copyright (C) 2008 by Pedro Mendes, Virginia Tech Intellectual </comment>
<comment> Properties, Inc., EML Research, gGmbH, University of Heidelberg, </comment>
<comment> and The University of Manchester. </comment>
<comment> All rights reserved. </comment>

 <comment> All rights reserved. </comment>
 <class>CQModelValue</class>
 <widget class="CopasiWidget" name="CQModelValue" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>394</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="caption" stdset="0" >
   <string>Global Quantity</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" >
   <item>
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <widget class="QLabel" name="mpLblName" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Name</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" >
      <widget class="QLabel" name="mpLblType" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Simulation Type</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="2" >
      <widget class="QComboBox" name="mpComboBoxType" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="0" >
      <widget class="QLabel" name="mpLblExpression" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Expression</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0" >
      <widget class="QLabel" name="mpLblInitialValue" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Initial Value</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="3" >
      <widget class="QCheckBox" name="mpBoxUseInitialExpression" >
       <property name="text" >
        <string>Use Initial Expression</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" >
      <widget class="QLabel" name="mpLblInitialExpression" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Initial Expression</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="4" >
      <widget class="Line" name="mpLine1" >
       <property name="frameShape" >
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item row="6" column="0" >
      <widget class="QLabel" name="mpLblValue" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Value</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="7" column="0" >
      <widget class="QLabel" name="mpLblRate" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Minimum" hsizetype="Preferred" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text" >
        <string>Rate</string>
       </property>
       <property name="alignment" >
        <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
       </property>
       <property name="wordWrap" >
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="9" column="0" colspan="4" >
      <widget class="Line" name="mpLine2" >
       <property name="frameShape" >
        <enum>QFrame::HLine</enum>
       </property>
       <property name="frameShadow" >
        <enum>QFrame::Sunken</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3" >
      <widget class="QLineEdit" name="mpEditName" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Fixed" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3" >
      <widget class="CQExpressionMmlStackedWidget" name="mpExpressionEMW" >
       <property name="minimumSize" >
        <size>
         <width>310</width>
         <height>60</height>
        </size>
       </property>
       <widget class="QWidget" name="page" />
       <widget class="QWidget" name="page_2" />
      </widget>
     </item>
     <item row="4" column="1" colspan="3" >
      <widget class="CQExpressionMmlStackedWidget" name="mpInitialExpressionEMW" >
       <property name="minimumSize" >
        <size>
         <width>310</width>
         <height>60</height>
        </size>
       </property>
       <widget class="QWidget" name="page_3" />
       <widget class="QWidget" name="page_4" />
      </widget>
     </item>
     <item row="3" column="1" colspan="2" >
      <widget class="QLineEdit" name="mpEditInitialValue" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="2" >
      <widget class="QLineEdit" name="mpEditRate" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="6" column="1" colspan="2" >
      <widget class="QLineEdit" name="mpEditCurrentValue" >
       <property name="sizePolicy" >
        <sizepolicy vsizetype="Preferred" hsizetype="MinimumExpanding" >
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="readOnly" >
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2" >
      <spacer name="mpSpacer" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Expanding</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="mpBtnCommit" >
       <property name="text" >
        <string>Commit</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnRevert" >
       <property name="text" >
        <string>Revert</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnNew" >
       <property name="text" >
        <string>New</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="mpBtnDelete" >
       <property name="text" >
        <string>Delete</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <customwidgets>
  <customwidget>
   <class>CopasiWidget</class>
   <extends>QWidget</extends>
   <header>copasiWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CQExpressionMmlStackedWidget</class>
   <extends>QStackedWidget</extends>
   <header>CQExpressionMmlStackedWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>mpComboBoxType</tabstop>
  <tabstop>mpBoxUseInitialExpression</tabstop>
  <tabstop>mpBtnCommit</tabstop>
  <tabstop>mpBtnRevert</tabstop>
  <tabstop>mpBtnNew</tabstop>
  <tabstop>mpBtnDelete</tabstop>
 </tabstops>
 <includes>
  <include location="global" >string</include>
  <include location="local" >copasiWidget.h</include>
  <include location="local" >CQExpressionMmlWidget.h</include>
  <include location="local" >CQExpressionMmlWidget.h</include>
 </includes>
 <resources/>
 <connections>
  <connection>
   <sender>mpBtnCommit</sender>
   <signal>clicked()</signal>
   <receiver>CQModelValue</receiver>
   <slot>slotBtnCommit()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnDelete</sender>
   <signal>clicked()</signal>
   <receiver>CQModelValue</receiver>
   <slot>slotBtnDelete()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnNew</sender>
   <signal>clicked()</signal>
   <receiver>CQModelValue</receiver>
   <slot>slotBtnNew()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBtnRevert</sender>
   <signal>clicked()</signal>
   <receiver>CQModelValue</receiver>
   <slot>slotBtnRevert()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpComboBoxType</sender>
   <signal>activated(int)</signal>
   <receiver>CQModelValue</receiver>
   <slot>slotTypeChanged(int)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpBoxUseInitialExpression</sender>
   <signal>toggled(bool)</signal>
   <receiver>CQModelValue</receiver>
   <slot>slotInitialTypeChanged(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mpEditName</sender>
   <signal>lostFocus()</signal>
   <receiver>CQModelValue</receiver>
   <slot>slotNameLostFocus()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel" >
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
